### DISCUSSION TITLE
Colour picker field using Vanilla Picker

### DISCUSSION THREAD

**Discussion by RVxLab** [filamentphp/filament#484] [Showcase]
I created a colour picker field by integrating [Vanilla Picker](https://github.com/Sphinxxxx/vanilla-picker).

It supports most options with the most notable one missing being `template`. I'm still working this one out, leaning towards being able to pass a view.

You can find the package here: https://github.com/RVxLab/filament-colorpicker

Please let me know what you think, any and all feedback is much appreciated!

**Comment by danharrin** [comment]
Looks great! Well done.
